src/shared/locales/app.js

```diff
@@ -46,6 +47,11 @@ const resources = {
       ...appLocalePtBr
     }
   },
+  'ru-RU': {
```

Comment:
I think this locale key is wrong, you should use `ru` instead of `ru-RU`.

https://electronjs.org/docs/api/locales

> ru | Russian